library/src/main/java/com/novoda/downloadmanager/lib/NotificationCustomiser.java

```diff
@@ -2,7 +2,55 @@
 
 import android.support.v4.app.NotificationCompat;
 
-public interface NotificationCustomiser {
+import com.novoda.downloadmanager.notifications.CancelledNotificationCustomiser;
+import com.novoda.downloadmanager.notifications.CompleteNotificationCustomiser;
+import com.novoda.downloadmanager.Download;
+import com.novoda.downloadmanager.notifications.DownloadingNotificationCustomiser;
+import com.novoda.downloadmanager.notifications.FailedNotificationCustomiser;
+import com.novoda.downloadmanager.notifications.QueuedNotificationCustomiser;
 
-    void modifyQueuedOrDownloadingNotification(NotificationCompat.Builder builder, long batchId);
+class NotificationCustomiser implements QueuedNotificationCustomiser, DownloadingNotificationCustomiser,
```

Comment:
we wanted to keep this class so that we can still pass it in to the notification displayer